SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -225,6 +227,59 @@ namespace ESpatialOSNetFlow
 	};
 }
 
+UENUM()
+namespace ESpatialOSRuntimeVariant
+{
+	enum Type
+	{
+		Standard,
+		CompatibilityMode
+	};
+}
+
+USTRUCT()
+struct SPATIALGDKEDITOR_API FRuntimeVariantVersion
+{
+	friend class USpatialGDKEditorSettings;
+	friend class FSpatialRuntimeVersionCustomization;
+
+	GENERATED_BODY()
+
+	FRuntimeVariantVersion() = default;
+
+	FRuntimeVariantVersion(const FString& InPinnedVersion) : PinnedVersion(InPinnedVersion)
+	{}
+
+	/** Returns the Runtime version to use for cloud deployments, either the pinned one, or the user-specified one depending on the settings. */
+	const FString& GetVersionForCloud() const;
+
+	/** Returns the Runtime version to use for local deployments, either the pinned one, or the user-specified one depending on the settings. */
+	const FString& GetVersionForLocal() const;
+
+	bool GetUseGDKPinnedRuntimeVersion() const { return bUseGDKPinnedRuntimeVersion; }
+
+	const FString& GetPinnedVersion() const { return PinnedVersion; }
+
+protected:
```

Comment:
Nit: Is this meant to be inherited? If not, and unless you have a future use case in mind this is for, I suggest changing to `private` for simplicity.

Reply:
Done!